src/components/CompareResults/SubtestsResults/SubtestsResultsTable.tsx

```diff
@@ -64,6 +64,21 @@ const cellsConfiguration: CompareResultsTableConfig[] = [
 
     gridWidth: '1fr',
   },
+  {
+    name: 'Base`σ`',
+    key: 'basestandarddeviation',
+    gridWidth: '1fr',
+  },
+  {
+    key: 'comparisonSignstandarddeviation',
+    gridWidth: '0.2fr',
+  },
+  {
+    name: 'New`σ`',
+    key: 'newstandarddeviation',
+
+    gridWidth: '1fr',
+  },
```

Comment:
Can you use camelCase for the naming of the keys.

Reply:
definitely